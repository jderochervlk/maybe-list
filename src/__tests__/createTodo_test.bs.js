// Generated by ReScript, PLEASE EDIT WITH CARE
'use strict';

var Jest = require("@glennsl/rescript-jest/src/jest.bs.js");
var React = require("react");
var CreateTodo = require("../CreateTodo.bs.js");
var ReactTestingLibrary = require("../external/ReactTestingLibrary.bs.js");

Jest.test("isEmpty when input is empty", (function (param) {
        return Jest.Expect.toBe(Jest.Expect.expect(CreateTodo.isEmpty("")), true);
      }));

Jest.test("isEmpty when input is not empty", (function (param) {
        return Jest.Expect.toBe(Jest.Expect.expect(CreateTodo.isEmpty("foo")), false);
      }));

Jest.test("Adding a todo", (function (param) {
        var handleCreate = function (_i) {
          
        };
        return Jest.Expect.toMatchSnapshot(Jest.Expect.expect(ReactTestingLibrary.render(undefined, undefined, undefined, undefined, undefined, React.createElement(CreateTodo.make, {
                                onCreate: handleCreate
                              })).container));
      }));

/*  Not a pure module */
