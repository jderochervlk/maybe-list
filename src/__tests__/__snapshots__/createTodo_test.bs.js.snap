// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adding a todo 1`] = `
<div>
  <div
    class="css-1239x0i"
  >
    <form>
      <div
        class="css-1n3bkpz"
      >
        <input
          class="chakra-input css-0"
          id="new-todo"
          placeholder="enter task name"
          value=""
        />
        <button
          class="chakra-button css-4xx2wk"
          disabled=""
          role="submit"
          type="submit"
        >
          add new task
        </button>
      </div>
    </form>
  </div>
</div>
`;
